src/styles/components/_component-grid.scss

```diff
@@ -11,14 +11,11 @@ $breakpoint-tablet: 820px;
     display: flex;
     justify-content: center;
     width: 100%;
-<<<<<<< HEAD
     margin: 0;
-    background-color: white;
+    background-color: var(--card-background-color);
     text-align: center;
     align-items: flex-start;
-=======
-    background-color: var(--background-color);
->>>>>>> a4258c0 (feat: add dark mode for text, logo, and borders)
+
```

Comment:
Your linter is mad about an extra line here, you can delete one of the lines of 18 or 19 and it should be all 😄 again.